packages/j/juliaup/package.yml

```diff
@@ -1,26 +1,22 @@
 name       : juliaup
-version    : 1.11.22
-release    : 1
+version    : 1.17.2
+release    : 2
 source     :
-    - https://github.com/JuliaLang/juliaup/archive/refs/tags/v1.11.22.tar.gz : 8a0b4540262a8bea9e61a4f287ced32ec4ad2ac4cb528a1b473eea6f139256df
+    - https://github.com/JuliaLang/juliaup/archive/refs/tags/v1.17.2.tar.gz : d0860697e70b5cf8c5ee8a529da82413317d996f3f1a34ff6a6fef3a4564bbd5
 homepage   : https://github.com/JuliaLang/juliaup
 license    : MIT
 component  : programming.tools
-summary    : Julia installer and version multiplexer 
+summary    : Julia installer and version multiplexer
 description: |
     A cross-platform installer for the Julia programming language.
 replaces   : julia
 networking : yes
 builddeps  :
     - rust
 setup      : |
-    # It seems like the `built` crate is having trouble fetching system
-    # information inside solbuild's container environment
-    sed -i 's|built::write_built_file().expect("Failed to acquire build-time information");||' build.rs
-    sed -i 's|include!(concat!(env!("OUT_DIR"), "/built.rs"))|pub const PKG_VERSION: \&str = "%version%"|' src/lib.rs
-    cargo fetch --locked
+    %cargo_fetch
 build      : |
-    cargo build --frozen %JOBS% --release --bin juliaup --bin julialauncher --features binjulialauncher
+    %cargo_build --bin juliaup --bin julialauncher --features binjulialauncher
 install    : |
-    install -Dm00755 target/release/{juliaup,julialauncher} -t $installdir/usr/bin
+    %cargo_install
```

Comment:
This change only installs `juliaup` and not `julialauncher`.

I think you'll have to add another macro call for it.

Reply:
Should work fine now with the newest `ypkg` (still needs a rebuild of course)

Reply:
Should work with `%cargo_install juliaup julialauncher` now with the newest `ypkg`